### ISSUE TITLE
Leanback Support (part 2)

### ISSUE THREAD

**Issue by bspinner** [MvvmCross/MvvmCross-AndroidSupport#137]
Leanback support is still far from finished. While one's already able to pull off a leanback app (yay!), there's still stuff to do.

For example:
- Proper bindings on most of the new fragments (at the moment you have to stuff your items into the Adapters/Presenters by hand)
- Support for Android's presenters support
- Caching for all new fragments

Currently I'm working on a leanback app, so I will gradually give back any emerged improvements, once they're cleaned up. And furthermore I'll refine this list...

**Comment by martijn00** [comment]
Do you have an update on this?

**Comment by bspinner** [comment]
Nope, haven't done anything on that app for some months now and therefore nothing regarding leanback.
